spec/models/user_spec.rb

```diff
@@ -135,6 +135,30 @@
     end
   end
 
+  describe '#validations' do
+    context 'password' do
+      let(:user) { create :user }
+
+      context 'with valid one' do
+        it 'saves without any errors' do
```

Comment:
These seem like they should integration tests if there's any persistence happening. You can also use `build :user` instead and check if `user.valid?` to avoid hitting the DB in unit tests

Reply:
The description for this test no longer matches what the test does here. We also need some feature specs around this functionality